core/src/main/java/io/kestra/core/storages/FileAttributes.java

```diff
@@ -0,0 +1,23 @@
+package io.kestra.core.storages;
+
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+
+@JsonSerialize(as = FileAttributes.class)
+public interface FileAttributes {
+    String getFileName();
+
+    long getLastModifiedTime();
+
+    long getCreationTime();
+
+    FileType getType();
+
+    long getSize();
+
+    enum FileType {
```

Comment:
I wonder if unknown and symbolicLink can have any meaning in our context, what will we do with them?

Reply:
probably nothing I could update to only support file and directory

Reply:
throw if it's neither a file or a directory